src/commands/slow-mode.ts

```diff
@@ -0,0 +1,579 @@
+		.setFields(
+			{
+				name: 'User',
+				value: `<@${interaction.user.id}>`
+			},
+			{
+				name: 'Channel',
+				value: `<#${channel.id}>`
+			}
+		).setFooter({
+			text: 'Pretendo Network',
+			iconURL: interaction.guild!.iconURL()!
+		});
+
+	await sendEventLogMessage(interaction.guild!, channel.id, auditLogEmbed);
+
+	await interaction.followUp({ content: `Slow mode disabled for <#${channel.id}>` });
+}
+
+async function slowModeStatsHandler(interaction: ChatInputCommandInteraction): Promise<void> {
+	await interaction.deferReply({ ephemeral: true });
+
+	const channel = interaction.options.getChannel('channel', false, [ChannelType.GuildText]) ?? interaction.channel;
+	if (!channel) {
+		throw new Error('No channel given');
+	}
+
+	if (channel.type !== ChannelType.GuildText) {
+		throw new Error('Slow mode only applies to text channels');
+	}
+
+	const slowMode = await SlowMode.findOne({
+		where: {
+			channel_id: channel.id
+		},
+		include: 'stages'
+	});
+
+	if (channel.rateLimitPerUser === 0) {
+		if (!slowMode || !slowMode.enabled) {
+			throw new Error(`No slow mode set for <#${channel.id}>`);
+		}
+	}
+
+	const embed = new EmbedBuilder()
+		.setTitle('Slow mode stats')
+		.setFields([
+			{
+				name: 'Channel',
+				value: `<#${channel.id}>`
+			},
+		])
+		.setFooter({
+			text: 'Pretendo Network',
+			iconURL: interaction.guild!.iconURL()!
+		});
+	
+	if (slowMode && slowMode.enabled) {
+		embed.addFields([
+			{
+				name: 'Participating Users',
+				value: slowMode.users.toString()
+			},
+			{
+				name: 'Current Rate',
+				value: slowMode.rate.toString()
+			},
+			{
+				name: 'Current Limit',
+				value: `1 message every ${slowMode.limit} seconds`
+			},
+			{
+				name: 'Window',
+				value: `${(slowMode.window / 1000).toString()} seconds`
+			}
+		]);
+
+		if (slowMode.stages) {
+			const stagesMessage = slowMode.stages.sort((a, b) => a.threshold - b.threshold).map(stage => {
+				return `1 message every ${stage.limit}s above ${stage.threshold} messages per minute`;
+			}).join('\n');
+		
+			embed.addFields([
+				{
+					name: 'Configured stages',
+					value: stagesMessage
+				}
+			]);
+		}
+	} else {
+		embed.addFields([
+			{
+				name: 'Current limit',
+				value: `1 message every ${channel.rateLimitPerUser} seconds`
+			}
+		]);
+	}
+
+	await interaction.followUp({ embeds: [embed] });
+}
+
+const command = new SlashCommandBuilder()
+	.setDefaultMemberPermissions('0')
+	.setName('slow-mode')
+	.setDescription('Configure slow mode for a channel')
+	.addSubcommandGroup(group => {
+		group.setName('enable')
+			.setDescription('Enable slow mode for a channel')
+			.addSubcommand(cmd => {
+				cmd.setName('auto')
+					.setDescription('Enable auto slow mode')
+					.addChannelOption(option => {
+						option.setName('channel')
+							.setDescription('The channel to enable auto slow mode for');
+						return option;
+					})
+					.addNumberOption(option => {
```

Comment:
I think that all of the `addNumberOption`s here should be `addIntegerOption`, as having a float for the window, threshold, or limit doesn't make sense.